core/domain/email_manager_test.py

```diff
@@ -6112,13 +6112,15 @@ class ContributionReviewerEmailTest(test_utils.EmailTestBase):
     TRANSLATION_REVIEWER_EMAIL: Final = 'translationreviewer@example.com'
     VOICEOVER_REVIEWER_EMAIL: Final = 'voiceoverreviewer@example.com'
     QUESTION_REVIEWER_EMAIL: Final = 'questionreviewer@example.com'
+    SUBMIT_QUESTION_EMAIL: Final = 'submitquestion@example.com'
 
     def setUp(self) -> None:
         super().setUp()
         self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
         self.signup(self.TRANSLATION_REVIEWER_EMAIL, 'translator')
         self.signup(self.VOICEOVER_REVIEWER_EMAIL, 'voiceartist')
         self.signup(self.QUESTION_REVIEWER_EMAIL, 'question')
+        self.signup(self.SUBMIT_QUESTION_EMAIL, 'questioner')
```

Comment:
```suggestion
        self.signup(self.QUESTION_CONTRIBUTOR_EMAIL, 'question-contributor')
```

Reply:
'question-contributor' is not a valid username , nor is 'question_contributor', so i have kept the username as 'questioncontributor'